quickbudget/views.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework import generics
+
+from quickbudget.api.serializers import ExpenseSerializer, \
+    CategoriesSerializer, BudgetListSerializer
+from quickbudget.models import Budget, Expense, Category
+
+
+class ListAddBudgets(generics.ListCreateAPIView):
+    queryset = Budget.objects.all()
+    serializer_class = BudgetListSerializer  # without expenses
+
+
+class BudgetDetail(generics.RetrieveUpdateDestroyAPIView):
+    queryset = Budget.objects.all()
```

Comment:
you need to be careful when usiing `queryset = Budget.objects.all()`. in some cases you'd need to overwrite `get_queryset` method and filter data.